### PR TITLE
Revamp kickback page

### PR DESCRIPTION
**Pull request by frankharkins**
## Changes

Reviewed the phase kickback page (see https://github.com/Qiskit/qiskit.org/issues/2437)

When merged, should close https://github.com/Qiskit/qiskit.org/issues/2437

## Implementation details

`ch-gates/phase-kickback` is now `algorithms/phase-kickback`

### CODE REVIEW

File: notebooks/algorithms/phase-kickback.ipynb [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+    "![](images/kickback/cnot-kickback.svg)\n",
+    "\n",
+    "Again, in this case the phase change $\\theta = 0.5$, so our control qubit is flipped around the Z-axis.\n",
+    "\n",
+    "<!-- ::: q-block -->\n",
+    "\n",
+    "### Worked example\n",
+    "\n",
+    "<details>\n",
+    "    <summary>Kickback with the CNOT gate(click to expand)</summary>\n",
```
Comment (techtolentino) [inline notebooks/algorithms/phase-kickback.ipynb:373]: Tiny missing space here `CNOT gate(click to expand)`

should be: `CNOT gate (click to expand)`

**Review by techtolentino — COMMENTED** [review]
@frankharkins - the smallest review and change request ever... just need a missing space. Otherwise, everything looks good to me.

**Comment by eddybrando** [comment]
Test URL in online preview: https://platypus-pr-752.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/algorithms/phase-kickback

**Review by eddybrando — APPROVED** [review]
Thank you very much @frankharkins! 🙂

**Pull request merged**